.husky/pre-commit

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npx lint-staged
-npm run build-types -- --local
-git add 'docs/*.api.md'
+#npx lint-staged
```

Comment:
those you should not add to the commit though...

Reply:
Well, that's not going to work then. I can of course deactivate the script, then commit, and the API docs won't be generated. But as soon as I activate the pre commit hook again, I have to commit so as to not push the deactivated script, and then the API docs will be regenerated with the blank lines.

Reply:
nothing forces you to add `.husky/pre-commit` to the commit. After the commit you only push...

Reply:
I thought that it'd be a good idea to perhaps fix the underlying "issue" , but yes, maybe some other time :)